release-notes/opensearch-knn.release-notes-2.7.0.0.md

```diff
@@ -0,0 +1,30 @@
+## Version 2.7.0.0 Release Notes
+
+Compatible with OpenSearch 2.7.0
+
+
+### Bug Fixes
+
+* Throw errors on model deletion failures ([#834](https://github.com/opensearch-project/k-NN/pull/834))
+
+### Infrastructure
+
+* Adding filter type to filtering release configs ([#792](https://github.com/opensearch-project/k-NN/pull/792))
```

Comment:
Nit: Could we change this to Add instead of Adding for consistency sake? This is a process we should follow when modifying change log

Reply:
Ack